Tests/test_Picture.py

```diff
@@ -30,6 +31,42 @@ def makeReferenceImage(filename='refimage.jpg'):
 
 # Testing functions
 
+def test_init():
```

Comment:
These tests look good. Just wondering if they are working well?

Reply:
They've been working for me -- do they work for you?  You'll need to install pytest...